### ISSUE TITLE
Cannot set imported empty column as Primary Key

### ISSUE THREAD

**Issue by AdrianPock** [sqlitebrowser/sqlitebrowser#574]
I added column titles '_id' and 'data' to the end of the first row of a csv file (no data in any row) hoping to make the '_id' column the primary key after importing the file.

The 'PK' status does not stick when modifying the table, unless I fill the column with numbers first.

Adding the column manually works fine (and the column is filled with numbers - perfect).

**Comment by chrisjlocke** [comment]
When modifying the table and the PK status doesn't stick, is the database reporting any error?  I presume the SQL log would show the commands sent and the errors (if any) returned?

**Comment by AdrianPock** [comment]
There are no errors shown in the log page.

However I think I have found the cause.

The file was created by saving from Excel so while there may be no data in the cells there are trailing commas on each row giving an 'empty' cell not a 'NULL' one. 

If I remove these commas the 'PK' status sticks and the column gets filled as expected.

Any chance of a "Add 'Primary Key' column" option in the csv import box?

Should I add a make separate request for this?

**Comment by justinclift** [comment]
Nah, here is probably good enough.  I'll just change the tag to "Enhancement" and we'll get to it whenever we do. :wink:
